### PR TITLE
Use Red rather than gray as record button backgnd when recording

### PR DESCRIPTION
**Pull request by aymanhab**
Avoid conflict with commonly used gray background

### CODE REVIEW

File: editor/js/OpenSimToolbar.js [hunk 1]
```diff
@@ -189,7 +189,7 @@ var OpenSimToolbar = function ( editor ) {
 	recordButton.onClick(function () {
 		recordStatus = !recordStatus;
 		if (recordStatus) // Dark gray if recording, should be done in CSS
```
Comment (aseth1) [inline editor/js/OpenSimToolbar.js]: Can you update the comment to indicate what color will be presented in the GUI?

**Comment by aymanhab** [comment]
Thanks for the catch @aseth1 here's how it would look like 
capturerecordred

**Comment by aymanhab** [comment]
Any feedback @aseth1 ? The comment has been updated. Thank you.

**Comment by tkuchida** [comment]
There will still be an issue if the background is red (or orange, pink, etc.). I think a better strategy might be to set the button's background to a color that is complementary to the Visualizer's background color. https://serennu.com/colour/rgbtohsl.php

**Comment by aymanhab** [comment]
Thanks @tkuchida, good idea.  While this would work if we have a blank background with fixed color, the issue is more complicated if we actually have a texture/Skybox. Adding a border maybe more appropriate general solution. If the color (bright red) is far from our default background(s) we provide then it likely would not happen by coincidence.

**Comment by tkuchida** [comment]
> While this would work if we have a blank background with fixed color, the issue is more complicated if we actually have a texture/Skybox.

Ah ok, good point.

**Comment by aymanhab** [comment]
Thanks much @tkuchida  👍

**Pull request merged**